### DISCUSSION TITLE
"Last modified" format option

### DISCUSSION THREAD

**Discussion by lerome** [filebrowser/filebrowser#1270] [Ideas]
It would be nice to have a setting allowing to choose between the current "relative" format and an absolute format (yyyy-MM-dd HH:mm:ss).

The current relative format is not precise enough for some use cases and/or for older files.

**Comment by ryjogo** [comment]
Second this! please implement, for my use case the item date/time is very important.

**Comment by junaidpv** [comment]
+1
Good to have this.

**Comment by junaidpv** [comment]
I figured out it is already possible to se that in profile settings:
<img width="705" height="489" alt="Screenshot from 2026-01-15 12-38-16" src="https://github.com/user-attachments/assets/d83017c4-8be3-4824-8029-fd6263b0e3bc" />

**Reply by lerome** [reply]
it was added 9 months after I opened the issue: https://github.com/filebrowser/filebrowser/pull/1536

**Comment by lerome** [comment]
https://github.com/filebrowser/filebrowser/pull/1536
